src/Symfony/Component/Routing/Loader/DependencyInjection/ServiceRouterLoaderContainer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Loader\DependencyInjection;
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * @internal
+ *
+ * @deprecated since Symfony 4.3, to be removed in 5.0
```

Comment:
`@internal, to be removed in 5.0`

Reply:
👌 but https://github.com/symfony/symfony/pull/30926#discussion_r272832533 ^^

Reply:
oups :) then let's add both!